docs/aerodromes/Albury (YMAY)/coordination.md

```diff
@@ -39,4 +39,10 @@ ML-BLA_CTR will coordinate all arrivals into Albury Prior to handing over to tow
 
 !!! example
     **ML-BLA_CTR** -> **AY_TWR**: "JST416, A320, estimates Albury 52, ARRAN1 arrival, 5000ft, number 1”
-    **AY_TWR** -> **ML-BLA_CTR**: "JST416"
\ No newline at end of file
+    **AY_TWR** -> **ML-BLA_CTR**: "JST416"
+
+## Standard Assignable Levels
+
+All aircraft departing from Albury shall be assigned `A060`.
```

Comment:
IRL this it is the lower of RFL or 7000FT for departure.
On descent assigned 8000FT prior to transfer (from memory).

Reply:
zach's levels are correct